### PR TITLE
data: im-machakata (init)

### PR DESCRIPTION
**Pull request by im-machakata**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Adding new profile data

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6007"></a>

### CODE REVIEW

File: data/EddieHubCommunity/im-machakata.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Isaac Machakata",
+  "type": "personal",
+  "bio": "Developer",
+
+  "tags": [
+    "Javascript",
+    "Typescript",
+    "VueJS",
+    "DevOps",
```
Comment (shyamtawli) [inline data/EddieHubCommunity/im-machakata.json:10]: remove `,` from last element that is `"DevOps"`

**Review by shyamtawli — CHANGES_REQUESTED** [review]

File: data/EddieHubCommunity/im-machakata.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  ],
+  "milestones": [
+    {
+      "title": "Developed CAIMS",
+      "date": "2022",
+      "icon": "FaGithub",
+      "color": "green",
+      "description": "Online School Managements System",
+      "url": "https://caims.caasi.co.zw"
+    },
```
Comment (Sindhuinti) [inline data/EddieHubCommunity/im-machakata.json:54]: Please remove the comma `,` here

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/EddieHubCommunity/im-machakata.json (hunk 2 above)
Comment (im-machakata) [reply data/EddieHubCommunity/im-machakata.json]: Done, thanks!

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please remove the `im-machakata` folder and keep your filename in the `data` directory only.
Let us know if you need any help updating that. 🙂

**Comment by im-machakata** [comment]
Sorry for the delay. I had to resort to using my phone. Please let me know if there are any more errors

**Comment by im-machakata** [comment]
> Please remove the `im-machakata` folder and keep your filename in the `data` directory only. Let us know if you need any help updating that. 🙂

Thanks I had forgotten to push it.

**Review by shyamtawli — COMMENTED** [review]
LGTM now.

**Review by SaraJaoude — APPROVED** [review]
looks good now

**Pull request merged**